t/basic.t

```diff
@@ -4,6 +4,23 @@ use Test::More;
 use Test::Mojo;
 
 my $t = Test::Mojo->new('Qi');
-$t->get_ok('/')->status_is(200)->content_like(qr/Mojolicious/i);
+$t->get_ok('/')->status_is(200);
+
+$t = $t->post_ok('/api/assets' => {Accept =>'application/json' , dataType => 'json'} => json => {tag => '000000A' , add_stamp => '2016-08-10 16:21:21'})
+    ->status_is(200);
```

Comment:
Great!  You're on the right track!  In this case, you have two tests: 1) did the POST request succeed and 2) did you get back a 200 response.  For true value, you actually want to expect a specific result; it's often not sufficient to say you got back a 200.  You'd want to inspect the body of the response.  With the request that you crafted in the post_ok, what response are expecting?  json?  Any particular keys or key value pairs?  Test for those.  This capability is covered in Test::Mojo.